composer.json

```diff
@@ -24,6 +24,9 @@
     "require": {
         "php": ">=5.3.0"
     },
+    "require-dev": {
+        "cs278/mktemp": "*"
```

Comment:
Could you remove this and add it on the fly on the test script?

Reply:
@williamdes Done.